bundle/jsky.app.ot/src/main/java/jsky/app/ot/tpe/gems/CandidateGuideStarsTableModel.java

```diff
@@ -162,13 +166,10 @@ public String getId() {
                 return "GEMS"; // see entry in ot.skycat.cfg - needed for GemsSkyObjectFactory
             }
         };
-        skycatTable.setCatalog(new SkycatCatalog(entry));
-        return skycatTable;
     }
 
     /**
-     * Returns a list of SkyObjects corresponding to the checked (or unchecked) rows in the table
-     * @param checked if true, return the checked rows (candidates), otherwise the unchecked (non-candidates)
+     * Returns a list of SideralTargets corresponding to the checked (or unchecked) rows in the table
```

Comment:
Right, the checked variable was always called with the false value se we could as well just remove it